src/test/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/DeleteOperationTest.java

```diff
@@ -235,6 +244,314 @@ public void deleteWithDynamic() {
       assertThat(result.status()).hasSize(1).containsEntry(CommandStatus.DELETED_COUNT, 1);
     }
 
+    @Test
```

Comment:
missing the test case where you would hit the maximum of retries and abort because of that

Reply:
It is there deleteWithDynamicRetryFailure, to show if maximum retries done it aborts and return error